charts/minio/templates/deployment.yaml

```diff
@@ -31,11 +36,11 @@ spec:
         {{- if .Values.minio.readinessProbe.enabled }}
         readinessProbe:
           httpGet:
-          {{- .Values.minio.readinessProbe.httpGet | toYaml | nindent 12 }}
-          {{- end }}
```

Comment:
Does this still work? I assume yes. but could you check?

Reply:
Jut tested now. It works